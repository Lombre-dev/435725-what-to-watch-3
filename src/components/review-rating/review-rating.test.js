import React from 'react';
import renderer from 'react-test-renderer';
import ReviewRating from './review-rating';

const VALUE = 0;
const LIMIT = 5;
const HANDLE_EVENT = () => {};

describe(`<ReviewRating />`, () => {

  it(`render should be match markup`, () => {

    const result = renderer
      .create(<ReviewRating
        value={VALUE}
        limit={LIMIT}
        onValueChange={HANDLE_EVENT}
      />)
      .toJSON();

    expect(result).toMatchSnapshot();
  });
});
