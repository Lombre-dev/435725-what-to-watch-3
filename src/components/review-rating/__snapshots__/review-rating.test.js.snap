// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ReviewRating /> render should be match markup 1`] = `
<div
  className="rating"
>
  <div
    className="rating__stars"
  >
    <input
      className="rating__input"
      defaultChecked={false}
      id="star-0"
      name="rating"
      onChange={[Function]}
      type="radio"
      value={1}
    />
    <label
      className="rating__label"
      htmlFor="star-0"
    >
      Rating 
      1
    </label>
    <input
      className="rating__input"
      defaultChecked={false}
      id="star-1"
      name="rating"
      onChange={[Function]}
      type="radio"
      value={2}
    />
    <label
      className="rating__label"
      htmlFor="star-1"
    >
      Rating 
      2
    </label>
    <input
      className="rating__input"
      defaultChecked={false}
      id="star-2"
      name="rating"
      onChange={[Function]}
      type="radio"
      value={3}
    />
    <label
      className="rating__label"
      htmlFor="star-2"
    >
      Rating 
      3
    </label>
    <input
      className="rating__input"
      defaultChecked={false}
      id="star-3"
      name="rating"
      onChange={[Function]}
      type="radio"
      value={4}
    />
    <label
      className="rating__label"
      htmlFor="star-3"
    >
      Rating 
      4
    </label>
    <input
      className="rating__input"
      defaultChecked={false}
      id="star-4"
      name="rating"
      onChange={[Function]}
      type="radio"
      value={5}
    />
    <label
      className="rating__label"
      htmlFor="star-4"
    >
      Rating 
      5
    </label>
  </div>
</div>
`;
